### PR TITLE
Betterer meta

### PR DESCRIPTION
**Pull request by zeftilldeath**
Further standardisation and neatening of the meta area... bit by bit. This PR pulls the info onto the same line across templates, for a more consistant visual appearance across the site. I haven't touched colour logic yet - this is just layout. In the process spotted some more legacy test stuff that I was able to to get rid of, so resulted in a lot of removed scss.

# Before
screen shot 2018-04-24 at 10 33 10

# After
screen shot 2018-04-24 at 10 33 39

screen shot 2018-04-24 at 10 36 45

### CODE REVIEW

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 1]
```diff
@@ -1084,6 +1021,12 @@ $quote-mark: 35px;
         font-style: normal;
     }
 
+    .meta__extras,
+    .meta__number:not(.u-h) + .meta__number,
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: `.meta__number:not(.u-h)` looks a bit unsafe – we shouldn't really be targetting helpers like this. if it's hidden, does it matter if a style is applied to it?

File: static/src/stylesheets/module/_social.scss [hunk 2]
```diff
@@ -183,64 +158,6 @@ category: Common
...
-            & {
-                content: 'share';
-            }
-        }
-    }
-}
-
 // Pintrest is inappropriate in certain sections
 
 .social__item--pinterest {
```
Comment (Joshua-David) [inline static/src/stylesheets/module/_social.scss]: Do you think this pinterest bit could be removed, probably need to delete the icon include in the markup though. I think we can hunt and kill all pinterest occurrences now (might be a separate PR)

Comment (zeftilldeath) [reply static/src/stylesheets/module/_social.scss]: Yep let's do it in a separate PR. I am going to do the same with save for later

Comment (Joshua-David) [reply static/src/stylesheets/module/_social.scss]: 👍

**Comment by ScottPainterGNM** [comment]
YAYYYYYYY

File: static/src/stylesheets/module/content-garnett/_content.global.scss [hunk 3]
```diff
@@ -12,6 +12,15 @@
 }
 
 .meta__numbers {
+    @include mq($from: leftCol, $until: wide) {
+        // Excude templates that have a full width meta
+        .content:not(.content--liveblog):not(.content--gallery):not(.content--interactive) .meta__extras:not(.meta__extras--crossword) & {
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_content.global.scss]: beyond the scope of this i know, so no problem with it in this PR, but we coudl really do with removing the need for these selectors, they're really brittle... 😰

**Review by sndrs — APPROVED** [review]
all that red ❤️

File: static/src/stylesheets/module/content-garnett/_content.global.scss (hunk 3 above)
Comment (zeftilldeath) [reply static/src/stylesheets/module/content-garnett/_content.global.scss]: Yep definitely! Adding to my list.
